stable/openvpn/values.yaml

```diff
@@ -18,6 +18,11 @@ resources:
   requests:
     cpu: 300m
     memory: 128Mi
+persistence: 
+  Enabled: true
+  StorageClass: generic
```

Comment:
This should be commented out so beta can be used when storage class is supplied (e.g. https://github.com/kubernetes/charts/blob/master/stable/chronograf/values.yaml#L15)